### ISSUE TITLE
Updating from 0.1.6 to 0.1.7 fails: nil pointer evaluating interface {}.queueGroup

### ISSUE THREAD

**Issue by christian-kreuzberger-dtx** [keptn-contrib/job-executor-service#201]
```
Error: UPGRADE FAILED: template: job-executor-service/templates/deployment.yaml:158:26: executing "job-executor-service/templates/deployment.yaml" at <.Values.distributor.config.queueGroup.enabled>: nil pointer evaluating interface {}.queueGroup
```

Quickfix:

Add `--set distributor.config.queueGroup.enabled=true`

Long-term fix: See https://github.com/keptn-contrib/prometheus-service/pull/276/commits/8f187f6a7e301bbbc711afa6d3b8d24fe4bae146

**Comment by christian-kreuzberger-dtx** [comment]
This has already been fixed in https://github.com/keptn-contrib/job-executor-service/commit/567b8b8bb615c0af6bb9fded8d4a0c8c678dcf9f, but still needs to be released

**Comment by RoadRunnr** [comment]
I have tested 0.1.8-next.3 and that seems to work fine

**Comment by christian-kreuzberger-dtx** [comment]
Fixed in 0.1.8 release.
